staging/src/k8s.io/client-go/tools/leaderelection/resourcelock/leaselock.go

```diff
@@ -105,6 +152,11 @@ func (ll *LeaseLock) Identity() string {
 	return ll.LockConfig.Identity
 }
 
+// StartSync starts the synchornization of local cache
+func (ll *LeaseLock) StartSync(stopCh <-chan struct{}) {
+	go ll.leaseInformer.Informer().Run(stopCh)
```

Comment:
so, we save API calls but we increase cpu usage and memory ... is it worth the trade off?

Reply:
It doesn't increase cpu usage since we compensate it with less GET calls. For memory, it only stores one Lease object, which is very minimal.

Reply:
fair enough